ingress/src/main/java/com/redhat/service/bridge/ingress/IngressServiceImpl.java

```diff
@@ -1,42 +1,25 @@
 package com.redhat.service.bridge.ingress;
 
-import java.util.ArrayList;
-import java.util.List;
-
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
 
-import com.redhat.service.bridge.ingress.api.exceptions.IngressException;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
 import com.redhat.service.bridge.ingress.producer.KafkaEventPublisher;
 
 import io.cloudevents.CloudEvent;
 
 @ApplicationScoped
 public class IngressServiceImpl implements IngressService {
 
-    private final List<String> deployments = new ArrayList<>();
+    @ConfigProperty(name = "event-bridge.bridge.id")
```

Comment:
The shard operator configures the Ingress application with the `bridgeId`. This application then adds the `bridgeId` to the metadata of the cloud events. 
The user does not have to specify the `bridgeId` anymore in the requests because the endpoint already identifies a unique bridge instance.

Reply:
@r00ta Can this and the `KafkaEventPublisher` be combined into a single class for now?

Reply:
Sure. The idea was to already structure the project for the future but np, I'll remove those redundant classes

Reply:
Done